githooks/pre-push

```diff
@@ -1,3 +1,16 @@
 #!/bin/sh
 
-./gradlew build integrationTest
+#!/bin/bash
```

Comment:
Is there a reason to have two shebangs here?

Reply:
Thank you, this was a copy-paste mistake. Removed the latter one.